aequilibrae/paths/linear_approximation.py

```diff
@@ -480,28 +459,32 @@ def execute(self):  # noqa: C901
             # Just need to create some arrays for cost
             c.graph.set_graph(self.time_field)
 
-            self.aons[c._id] = allOrNothing(c.matrix, c.graph, c._aon_results)
+            self.aons[c._id] = allOrNothing(c._id, c.matrix, c.graph, c._aon_results)
 
+        self.equilibration.emit(["start", self.max_iter, "Equilibrium Assignment"])
         self.logger.info(f"{self.algorithm} Assignment STATS")
         self.logger.info("Iteration, RelativeGap, stepsize")
         for self.iter in range(1, self.max_iter + 1):  # noqa: B020
             self.iteration_issue = []
-            if pyqt:
-                self.equilibration.emit(["rgap", self.rgap])
-                self.equilibration.emit(["iterations", self.iter])
+            self.equilibration.emit(["key_value", "rgap", self.rgap])
+            self.equilibration.emit(["key_value", "iterations", self.iter])
 
             aon_flows = []
 
             self.__maybe_create_path_file_directories()
 
             for c in self.traffic_classes:  # type: TrafficClass
+                if self.assignment.pbar is None:
```

Comment:
```suggestion
                if self.assignment.pbar is not None:
```

??

Reply:
This is correct. When the pbar is None, a "start" signal starts it.  This would break with QAequilibraE, so I made a slight change.